client.js

```diff
@@ -66,6 +66,31 @@ client.prototype = {
     });
   },
 
+  enterPublic: function() {
```

Comment:
`enterPublic` to me semantically sounds like you're entering an existing room since `enterWithKey ` is used to join an existing room. I wonder if it would be better to make `reserve` take an `options` object which could define whether the room is `public` or not initially. I still think the best thing would be to create a new "lobby" type object that could be entered either by passing nothing (main lobby) or by passing in a lobby name. Then two users could be moved into a private room when needed.